int64 X
int64 Y
string Label
---
bool return_value